swift-ui-base/Common/View/TextField/View/TextFieldView.swift

```diff
@@ -39,24 +33,23 @@ struct TextFieldView: View {
           TextField("", text: $fieldData.value)
             .foregroundColor(.darkGray)
             .autocapitalization(.none)
-            .opacity(0.8)
             .accessibility(identifier: "\(fieldData.title.withNoSpaces)TextField")
         }
       }
       
       Rectangle()
         .frame(maxWidth: .infinity, maxHeight: 1)
         .foregroundColor(
-          !fieldData.isValid && fieldData.hasTyped ? .errorRed : .lightGray
+          !fieldData.isEmpty && !fieldData.isValid ? .errorRed : .lightGray
         )
         .opacity(0.5)
       
-      if !fieldData.isValid && fieldData.hasTyped {
-        Text(fieldData.errorMessage)
-          .frame(maxWidth: .infinity, alignment: .leading)
-          .font(.footnote)
-          .foregroundColor(.errorRed)
-      }
+      Text(fieldData.errorMessage)
+        .frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
wouldn't .infinity make the text to overflow if its long enough?

Reply:
Yes, unles you have a padding removing space from the sides and that's the case of the wrapping stack

Reply:
And how's the actual text behavior in that case, it would just clip it or we must add some modifier to the Text to show elipsis or similar?

Reply:
Sincerely, I can even imagine your question on my mind right now. Let me try variations on code and I can answer you later

Reply:
@glm4, behavior for long string:

<img width="338" alt="Screen Shot 2020-07-27 at 1 54 15 PM" src="https://user-images.githubusercontent.com/1467868/88569330-ad135180-d010-11ea-8761-fd1c823b62ac.png">

Reply:
cool!